apinotes/Foundation.apinotes

```diff
@@ -1738,3 +1738,13 @@ Enumerators:
   SwiftName: regularExpression
 - Name: NSStringEnumerationByLines
   SwiftName: byLines
+SwiftVersions:
+- Version: 3
+  Protocols:
+  - Name: NSFastEnumeration
+    Methods:
+    - Selector: 'countByEnumeratingWithState:objects:count:'
+      MethodKind: Instance
+      Parameters:
+      - Position: 1
+        Nullability: U
```

Comment:
Effectively yes. Clang dropped the annotation on the floor, which I fixed in apple/swift-clang#43. Landing that commit immediately started picking up the existing annotations in the Xcode 8 SDKs, but there are actually very few of those, and most of them are _Nullable rather than _Nonnull, which is source-compatible for calls.